opentelemetry-sdk/src/metrics/internal/aggregate.rs

```diff
@@ -12,7 +12,7 @@ use super::{
     precomputed_sum::PrecomputedSum, sum::Sum, Number,
 };
 
-const STREAM_CARDINALITY_LIMIT: u32 = 2000;
+pub(crate) const STREAM_CARDINALITY_LIMIT: u32 = 2000;
```

Comment:
nit: Could we use `usize` to avoid this type casting in `ValueMap`: `STREAM_CARDINALITY_LIMIT as usize`?

```suggestion
pub(crate) const STREAM_CARDINALITY_LIMIT: usize = 2000;
```